packages/react-vapor/src/components/calendar/Calendar.tsx

```diff
@@ -257,21 +269,42 @@ export class Calendar extends React.Component<ICalendarProps, any> {
             'calendar',
             {
                 'mod-width-50': !this.props.simple,
+                'countdown-calendar': this.props.countdown,
             },
             this.props.wrapperClassNames
         );
 
+        if (this.props.countdown) {
+            this.countdownHeader = (
+                <div id="countdown-header">
+                    <h2 className="bold mb1">
+                        {moment().date(moment().month(this.props.months[selectedMonth]).daysInMonth()).fromNow(true)}{' '}
+                        left
+                    </h2>
+                    <div className="smaller">in {this.props.months[selectedMonth]}</div>
+                </div>
+            );
+        }
+
+        const defaultHeader = (
+            <div className="calendar-header p2">
+                {monthPicker}
+                {yearPicker}
+            </div>
+        );
+
         return (
             <div className={wrapperClasses}>
-                <div className="calendar-header p2">
-                    {monthPicker}
-                    {yearPicker}
-                </div>
+                {this.props.showHeader && (this.countdownHeader || defaultHeader)}
```

Comment:
```suggestion
                {this.props.showHeader && (this.countdownHeader ?? defaultHeader)}
```

Reply:
> 

Whats the advantage of this if there's no preceding nullish check? Wouldn't they do the same thing?